src/schema/sketch-121/objects/object-id.ts

```diff
@@ -0,0 +1,11 @@
+import { brand, Branded, string } from 'io-ts';
+
+const UUIDReg = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/;
```

Comment:
Could we use `UUID` from `io-ts`/`io-ts-types`?

Reply:
Do you mean use UUID regex from io-ts lib? Unfortunately io-ts lib doesn't export this regex, and existing UUID codec from this lib isn't valid for sketch ids.

Reply:
ok, let's use custom one